client/telemetry/src/worker/node.rs

```diff
@@ -110,14 +114,20 @@ where TTrans: Clone + Unpin, TTrans::Dial: Unpin,
 	///
 	/// After calling this method, you should call `poll` in order for it to be properly processed.
 	pub fn send_message(&mut self, payload: impl Into<BytesMut>) -> Result<(), ()> {
-		if let NodeSocket::Connected(NodeSocketConnected { pending, .. }) = &mut self.socket {
-			if pending.len() <= MAX_PENDING {
+		if let NodeSocket::Connected(connec) = &mut self.socket {
+			if connec.pending.len() <= MAX_PENDING {
 				trace!(target: "telemetry", "Adding log entry to queue for {:?}", self.addr);
-				pending.push_back(payload.into());
+				connec.pending.push_back(payload.into());
 				Ok(())
 			} else {
-				warn!(target: "telemetry", "⚠️  Rejected log entry because queue is full for {:?}",
-					self.addr);
+				match connec.latest_pending_full_warning {
+					Some(m) if m.elapsed() < Duration::from_secs(15) => {}
```

Comment:
I'm a bit concerned that if that legitimately raises many errors now, we won't notice because we are dropping them and thus cover up a potential other issue. How about counting the number of messages we've dropped and include that in the message?